### PR TITLE
Add sim plugin

### LINKED ISSUE #306: Simcard
Any chance we could get support for https://github.com/pbakondy/cordova-plugin-sim or similar plugin?

Thank you for your work!

### PR DESCRIPTION
**Pull request by biesbjerg**
Closes #306

### CODE REVIEW

File: src/plugins/sim.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ *
+ * Sim.getSimInfo().then(
+ *   (info) => console.log('Sim info:', info),
+ *   (err) => console.log('Unable to get sim info:', err)
+ * );
+ * ```
+ */
+@Plugin({
+  plugin: 'cordova-plugin-sim',
+  pluginRef: 'window.plugins.sim',
```
Comment (ihadeed) [inline src/plugins/sim.ts]: this can be only `plugins.sim`

Ionic native automatically adds `window.`

**Comment by ihadeed** [comment]
Looks good to me! Just edit the pluginRef and it will be perfect!

Thanks for the PR @biesbjerg

**Comment by biesbjerg** [comment]
Done!

**Comment by ihadeed** [comment]
Thanks again @biesbjerg

**Pull request merged**